backend/src/app.py

```diff
@@ -111,25 +111,28 @@ def add_user():
             return jsonify({"success": False, "msg": "User does not exist"}), 204
 
     ## Account endpoints ##
-    @app.route("/account", methods=["POST"])
-    def add_account():
+    @app.route("/account", methods=["POST", "GET", "DELETE"])
+    def submit_account():
         if request.method == "POST":
             account_info = request.json
+        elif request.method == "GET" or request.method == "DELETE":
+            account_info = request.args
 
-            # Error check
+        with Session.begin() as session:
             if string_blank(account_info["username"]):
                 return jsonify({"response": "username cannot be blank"}), 400
-            if not account_type_valid(account_info["account_type"]):
-                return jsonify({"response": "Account type is not valid"}), 400
+            if not user_exists(session, account_info["username"]):
+                return (
+                    jsonify({"success": False, "msg": "User does not exist"}),
+                    400,
```

Comment:
Consider using a 422 status code here as this request is formed properly but can't be processed due to a non-existing user https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

Reply:
will update status codes in future as needed for clarity